lib/matplotlib/tests/test_figure.py

```diff
@@ -367,7 +367,7 @@ def test_subplots_shareax_loglabels():
 def test_savefig():
     fig = plt.figure()
     msg = "savefig() takes 2 positional arguments but 3 were given"
```

Comment:
```suggestion
    msg = "savefig\(\) takes 2 positional arguments but 3 were given"
```

Need to escape since it‘s a regexp.

Reply:
Yes, right, fixed. Though it also needed to be a raw string too.